### PR TITLE
Feature/add basic views

### PR DESCRIPTION
**Pull request by frozenhelium**
Depends on: https://github.com/toggle-corp/react-store/pull/596/
Depends on: https://github.com/toggle-corp/react-store/pull/599

### CODE REVIEW

File: react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    key: string | number;
+    value: number;
+    label: string | number;
+    color: string;
+    startAngle: number;
+    endAngle: number;
+}
+
+const arcRenderKeySelector = (datum: RenderDatum) => (
+    datum.key as string
```
Comment (tnagorra) [inline react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx]: Why cast to string?

File: react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    }
+
+    public render() {
+        const {
+            className,
+            boundingClientRect: {
+                width = 0,
+                height = 0,
+            },
+            data = [],
```
Comment (tnagorra) [inline react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx]: Instantiation of new array in render

File: react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx]: Because key selector does not accept number

Comment (tnagorra) [reply react/src/Root/App/Information/GeohazardAffectedHouseholds/DonutChart/index.tsx]: Will be fixed in https://github.com/toggle-corp/react-store/pull/599

**Pull request merged**